bin/tictactoe

```diff
@@ -1,3 +1,25 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "**** Welcome to Tic Tac Toe! ****\n\n"
+help
+
+loop do
+  puts "\nWhat mode do you want to play?"
+
+  case gets.strip
+  when "1"
+    zero_player
+  when "2"
+    one_player
+  when "3"
+    two_players
+  when "help"
+    help
+  when "exit"
+    break
+  else
+    puts "Invalid! type 'help' for help.\n\n"
+  end
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.